_data/contributions.yml

```diff
@@ -175,3 +175,6 @@ devs:
   - github_url: "https://github.com/kalamell"
     name: "Bundit S"
     avtar: "https://avatars2.githubusercontent.com/u/3000196?v=4"
+  - github_url: "https://github.com/lunecia"
+    name: "Lubia"
+    avtar: "https://avatars3.githubusercontent.com/u/12728959?s=400&v=4"
```

Comment:
Looks like something wrong, there is spell mistake in key, like avtar instead of avatar.